src/services/pscVerificationService.ts

```diff
@@ -21,6 +22,17 @@ export const createPscVerification = async (request: Request, transaction: Trans
     if (!sdkResponse) {
         throw createAndLogError(`${createPscVerification.name} - PSC Verification POST request returned no response for transaction ${transaction.id}`);
     }
+
+    if (sdkResponse.httpStatusCode === HttpStatusCode.InternalServerError) {
+        const apiErrorResponseResource = sdkResponse as ApiErrorResponse;
+        const apiErrorResponse = apiErrorResponseResource.errors?.[0] as ApiErrorResponse;
+        const error = apiErrorResponse.errors?.[0].errorValues?.error as string;
+
```

Comment:
this could be refactored down to 
`const error = (sdkResponse as ApiErrorResponse)?.errors?.[0]?.errorValues?.error as string;`

Reply:
inline variables done